.rustfmt.toml

```diff
@@ -0,0 +1,2 @@
+use_small_heuristics = "Max"
```

Comment:
This looks reasonable, but is there any particular reason to prefer it over the default?